mail_activity_team/static/src/components/activity_menu_view/activity_menu_view.xml

```diff
@@ -25,7 +21,7 @@
                         <a
                             class="team_activities o_filter_nav_item nav-link"
                             data-filter='team'
-                            t-on-click="activityMenuView.onClickActivityFilter"
+                            t-on-click="() => this.onClickActivityFilter('team_activities')"
```

Comment:
When keep the mouse over this menu (and  My Activities as well), the cursor should be "pointer"